pkgs/by-name/wp/wpewebkit/package.nix

```
@@ -0,0 +1,150 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  cmake,
+  ninja,
+  pkg-config,
+  perl,
+  python3,
+  ruby,
+  bison,
+  flex,
+  gperf,
+  gettext,
+  gobject-introspection,
+  glib,
+  gi-docgen,
+  libwpe,
+  libwpe-fdo,
+  libsoup_3,
+  libxml2,
+  libxslt,
+  libavif,
+  libgbm,
+  libjxl,
+  lcms2,
+  harfbuzz,
+  freetype,
+  icu,
+  libjpeg,
+  libpng,
+  sqlite,
+  zlib,
+  libepoxy,
+  mesa,
+  at-spi2-core,
+  libgcrypt,
+  libgpg-error,
+  libxkbcommon,
+  systemd,
+  libsysprof-capture,
+  libtasn1,
+  p11-kit,
+  libedit,
+  libwebp,
+  woff2,
+  bubblewrap,
+  libseccomp,
+  xdg-dbus-proxy,
+  gst_all_1,
+  unifdef,
+  wayland,
+  wayland-protocols,
+  wayland-scanner,
+  libdrm,
+  cairo,
+  fontconfig,
+  cctools,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wpewebkit";
+  version = "2.50.0";
+
+  src = fetchurl {
+    url = "https://wpewebkit.org/releases/wpewebkit-${version}.tar.xz";
+    sha256 = "sha256-qa9ixeGFUbc4a324ZOi6gVbyGbjmxjmTS/bzpWeWmSI=";
```

Use hash.  You may need to regenerate the hash.

```suggestion
    hash = "...";
```